### PR TITLE
Clear cookies when switching accounts

### LINKED ISSUE #8005: "Log in as user" feature does not reset printdisabled cookie
Logging in as a user via the admin UI for debugging does not reset the printdisabled cookies, causing some weird issues.

### Steps to Reproduce
1. Log in with admin, printdisabled account
2. "Log in as user" of another user account, not printdisabled accout

* Actual: pd=1 still set in cookies
* Expected: pd should be set to what makes sense for the user.

### Details

- **Logged in (Y/N)?**
- **Browser type/version?**
- **Operating system?**
- **Environment (prod/dev/local)?** prod

### PR DESCRIPTION
**Pull request by ClementineAccount**
This closes #8005.

This follows the suggestions proposed by Mek in https://github.com/internetarchive/infogami/pull/215#discussion_r1379004747
by creating handlers to call a new function that clears the cookies "pd" and "sfw"

### Technical

This implementation solves the issue mentioned by clearing the cookies using a handler to infogami, overriding infogami `logout` function. It also does this when 'logging in' as another user from the admin panel of `admin/people`

### Testing

(The video below under Screenshot also shows this step)
1. Login as an admin and observe the 'pd' cookie be created through the inspector
2. Logout as an admin and observe the 'pd' cookie has been cleared.
3. Login as admin again and observe that the pd cookie has been created again.
4. Login as another user using `admin/people` and observe that the pd cookie has been cleared.

### Screenshot

I recorded this video showcasing the 'pd' cookie being cleared when the hotfix is applied. Prior to the hotfix application, the pd cookie will never be cleared.

https://github.com/internetarchive/openlibrary/assets/26779639/79aa4738-f57c-496d-a5d0-21e6a592d8d4

### Stakeholders

@mekarpeles proposed the fix concept and suggested this issue to me as a 'Good First Issue' so he would likely want to review my approach prior to a merge.

### CODE REVIEW

**Comment by ClementineAccount** [comment]
I failed one of the CI/CD tests by accidentally importing infogami_logout twice. I have since fixed that but need to run the workflow again.

File: openlibrary/plugins/upstream/account.py [hunk 1]
```diff
@@ -478,6 +479,23 @@ def has_borrowed_at_least(self, amount: int, s3_keys) -> bool:
...
+
+    """
+
+    path = "/account/logout"
+
+    def POST(self):
+        clear_cookies()
+        from infogami.core.code import logout as infogami_logout
+
+        infogami_logout().POST()
```
Comment (mekarpeles) [inline openlibrary/plugins/upstream/account.py:496]:
```suggestion
        return infogami_logout().POST()
```

**Comment by mekarpeles** [comment]
provisional LGTM, great job -- will state on testing and give it a try

**Comment by cdrini** [comment]
Taking off testing to test #8550 .

**Review by jimchamp — APPROVED** [review]
This looks to be working as expected. Thanks, @ClementineAccount!

**Pull request merged**